### ISSUE TITLE
Thumb Sign

### ISSUE THREAD

**Issue by samardeep331** [obophenotype/human-phenotype-ontology#11208]
**Preferred term label:**
Thumb Sign

**Synonyms**
Thumb Sign (Ghent Criteria)

**Definition (free text, please give PubMed ID)**
The thumb sign is positive when the entire distal phalanx of the adducted thumb extends beyond the ulnar border of the palm with or without the assistance of the patient or examiner to achieve maximal adduction.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Marfan Syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001274
- label: Thumb sign
- parents: [Abnormal finger morphology (HP:0001167)](https://hpo.jax.org/app/browse/term/HP:0001167)
- pmid: PMID:7616992
- def: The distal phalanx of the adducted thumb extends beyond the ulnar border of the palm (The thumb protrudes from the clenched fist).
- comment: The Steinberg or the thumb sign is elicited by asking the patient to fold his thumb into the closed fist. If the thumb tip extends from the palm of the hand, the test is positive.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11208
- synonyms: Steinberg sign(NONE)
